tests/scripts/all.sh

```diff
@@ -608,6 +633,19 @@ make
 msg "test: ssl-opt.sh, MFL-related tests"
 if_build_succeeded tests/ssl-opt.sh -f "Max fragment length"
 
+# ... and with MFL disabled and small output buffer
```

Comment:
Please make the comment self-contained. You never know when this item is going to be moved, or some other item is going to be inserted before it.